examples/claim_and_new_game.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Used after running a server for the first time (or after clearing saved data: https://satisfactory.wiki.gg/wiki/Dedicated_servers/Configuration_files#Overview)
+# Claims the server with the provided name and admin password, prints the authentication token created, and creates a 
+# new game on the server with a random starting location, printing a json object with server info afterwards
+# Options can also be set with comma separated key=value
+
+from pyfactorybridge import API
+from pyfactorybridge.exceptions import ServerError
+from argparse import ArgumentParser
+from json import dumps
+from sys import exit, stderr
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument("password", help="the desired server admin password")
+    parser.add_argument("address", default="localhost:7777", nargs="?", help="the address of the server to connect to")
+    parser.add_argument("--token", help="If server is already claimed, specifies the authentication token to be used")
+    parser.add_argument("--server-name", default="My First Server", help="Sets the Server Name (shown in the Server Manager list). Defaults to 'My First Server'")
+    parser.add_argument("--session-name", default="My First Game", help="Sets the Session Name (shown in the Server Status screen after selecting the server to join). Defaults to 'My First Game'")
+    parser.add_argument("--options", help="Sets server options before starting the game (comma separated key=value). e.g: FG.DSAutoPause=false")
+    args = parser.parse_args()
+
+    token = args.token
+    if not token:
+        s = API(address=args.address)
```

Comment:
Could we have more descriptive variable names throughout?

Reply:
Changed `s` to `server`